### ISSUE TITLE
Added synch batch norm

### ISSUE THREAD

**Issue by edizhuang** [yassouali/pytorch_segmentation#17]
Hi Yassine,

The file related to synch batch norm seems to be changed, so does the requirement for config file. What does the new config need related to this change?

**Comment by yassouali** [comment]
Hi, there is a new option "use_synch_bnorm", it is False by default, set it ti true if you want to aggregate the gradients from the multiple Gpus and then update the BN patameters.

for the requirement file, just ignore it, the change is just a new PIL version.
